openreview/conference/templates/recruitResponseWebfield.js

```diff
@@ -26,12 +26,17 @@ function render() {
       // Display response text
       var message = 'Thank you for accepting this invitation from ' + HEADER.title;
       $response.append('<div class="panel"><div class="row"><h3 style="line-height:normal;">' + message + '</h3></div></div>');
+      var email = args.user.indexOf('@') > -1 ? '(<strong>' + args.user + '</strong>) ' : '';
 
       $response.append([
         '<div class="panel">',
           '<div class="row">',
-            '<p>If you do not already have an OpenReview account, please sign up <a style="font-weight:bold;" href="/signup">here</a>.</p>',
-            '<p>If you have an existing OpenReview account, please ensure that the email address that received this invitation is linked to your <a style="font-weight:bold;" href="/profile?mode=edit">profile page</a> and has been confirmed.</p>',
+            '<h4>Please complete the following steps now:</h4>',
+              '<ol>',
+                '<li><p>Log in to your OpenReview account (if you do not already have an OpenReview account, you can sign up <a style="font-weight:bold;" href="/signup">here</a>).</p></li>',
```

Comment:
I would make this 2 sentences. "Log in to your OpenReview account. If you do not already have an account, sign up for OpenReview here."

Reply:
Done. I have split it to "Log in to your OpenReview account. If you do not already have an account, you can sign up here."